mobile-browser-based-version/src/Task Definition/lus_covid.js

```diff
@@ -0,0 +1,250 @@
+import * as tf from '@tensorflow/tfjs';
+
+const FEATURES = 1000
+const MOBILENET_V1_1_PATH = 'https://storage.googleapis.com/tfjs-models/tfjs/mobilenet_v1_1.0_224/model.json'
+let net = null
+
+export class LusCovidTask {
+    constructor() {
+        this.display_information = display_information
+        this.training_information = training_information
+    }
+
+    /**
+     * fetch model from local storage or indexdb
+     * 
+     * @returns Returns a tf.model or null if there is no model
+     */
+     async get_model_from_storage() {
+        let model = await tf.loadLayersModel('indexeddb:://working_'.concat(training_information.model_id))
+        return model
+    }
+
+    /**
+     * @returns new instance of TensorflowJS model
+     */
+    create_model() {
+        const save_path_db = "indexeddb://working_".concat(
+            training_information.model_id
+        );
+        // only keep this here
+        this.create_deep_learning_model().save(save_path_db);
+    }
+
+    create_deep_learning_model(){
```

Comment:
I think this can be moved in the create_model function, it is a bit confusing to have create_model and create_deep_learning_model functions, it looks like you can create two types of models